### PR TITLE
Feature/team

### PR DESCRIPTION
**Pull request by garrettgregor**
This will update with a Teams class to perform additional statistics. All tests passing and formatting passes rubocop.

### CODE REVIEW

File: lib/game.rb
Comment (MylesNottingham) [inline lib/game.rb:1]: Like the spacing here. It's pretty!

File: lib/stat_tracker.rb
Comment (MylesNottingham) [inline lib/stat_tracker.rb:1]: Thanks for deleting the extra line.

**Review by MylesNottingham — APPROVED** [review]
Excellent

**Review by javenb022 — APPROVED** [review]
Loving it

**Pull request merged**